### PR TITLE
Adding additional targets for individual component images

### PR DESCRIPTION
**Pull request by ewolinetz**

### CODE REVIEW

**Comment by ewolinetz** [comment]
Adding valid bugzilla label since this is only affects building images for dev

File: Makefile [hunk 1]
```diff
@@ -14,3 +21,49 @@ test-upgrade:
...
+
+build-elasticsearch-image:
+	hack/build-component-image.sh "elasticsearch" $(ELASTICSEARCH_IMAGE)
+.PHONY: build-elasticsearch-image
+
+build-curator-image:
+	hack/build-component-image.sh "curator" $(CURATOR_IMAGE)
+.PHONY: build-curator-image
+
+build-all-images:
```
Comment (vimalk78) [inline Makefile:41]: this needs to change to :
```
build-all-images: build-fluentd-image build-kibana-image build-elasticsearch-image build-curator-image

```
please check master version

File: Makefile [hunk 2]
```diff
@@ -14,3 +21,49 @@ test-upgrade:
...
+
+deploy-elasticsearch-image: build-elasticsearch-image
+	hack/deploy-component-image.sh $(ELASTICSEARCH_IMAGE)
+.PHONY: deploy-elasticsearch-image
+
+deploy-curator-image: build-curator-image
+	hack/deploy-component-image.sh $(CURATOR_IMAGE)
+.PHONY: deploy-curator-image
+
+deploy-all-images:
```
Comment (vimalk78) [inline Makefile:64]: this needs to change to
```
deploy-all-images: deploy-fluentd-image deploy-kibana-image deploy-elasticsearch-image deploy-curator-image
```

**Comment by ewolinetz** [comment]
@vimalk78 addressed

**Review by igor-karpukhin — APPROVED** [review]
/lgtm

**Comment by jcantrill** [comment]
@ewolinetz can this PR be closed?  otherwise we may need to consider creating a bz to address

**Pull request closed without merging**